api/server/router/container/container_routes.go

```diff
@@ -465,6 +465,16 @@ func (s *containerRouter) postContainersCreate(ctx context.Context, w http.Respo
 		hostConfig.AutoRemove = false
 	}
 
+	// When using API 1.39 and under, BindOptions.NoRecursive should be ignored because it
```

Comment:
Are we doing this sort of change on other fields?

Reply:
https://github.com/moby/moby/pull/38003#discussion_r224488064